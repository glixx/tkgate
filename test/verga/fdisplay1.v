module top;

  integer f,g;
  reg x;

  initial
    begin
      f = $fopen("test1.out");
      g = $fopen("test2.out");
      $display("f=%d",f);
      $fdisplay(1|f|g|16,"hello world");
      x = 0;
      # 1;
      $fstrobe(1|f|g,"x=%d",x);
      $fdisplay(1|f|g,"x=%d",x);
      x = 1;
      # 1;
      $fmonitor(1|f,"%d: monitored x = %d", $time, x);
      # 1;
      x = 0;
      # 3;
      x = 1;
      # 2;
      x = 0;
    end

endmodule
