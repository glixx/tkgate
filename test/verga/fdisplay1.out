f=2
hello worldRuntime Error at 0: Write to closed descriptor.

Runtime Error at 0: Write to closed descriptor.
x=0
x=1
3: monitored x = 0
6: monitored x = 1
8: monitored x = 0
